src/widget/contentlayout.cpp

```diff
@@ -71,15 +71,11 @@ void ContentLayout::clear()
     QLayoutItem* item;
     while ((item = mainHead->layout()->takeAt(0)) != 0)
     {
-        item->widget()->hide();
-        item->widget()->setParent(nullptr);
```

Comment:
![spectacle m10381](https://cloud.githubusercontent.com/assets/6306608/17831390/4c6c7c88-66f8-11e6-80bc-edea80a25a86.png)

You disagree, that every commit should be a correct version? If not -- no question anymore